package.json

```diff
@@ -27,83 +27,84 @@
         "lint": "eslint . --ext js,mjs,jsx,ts,mts,tsx --max-warnings 0"
     },
     "dependencies": {
-        "autosuggest-highlight": "^3.2.0",
-        "clsx": "^1.0.4",
-        "jwt-decode": "^3.0.0",
+        "autosuggest-highlight": "^3.3.4",
+        "clsx": "^2.1.0",
+        "jwt-decode": "^4.0.0",
         "memoize-one": "^6.0.0",
-        "oidc-client": "^1.10.1",
-        "prop-types": "^15.7.2",
-        "react-csv-downloader": "^2.7.0",
+        "oidc-client": "^1.11.5",
+        "prop-types": "^15.8.1",
+        "react-csv-downloader": "^3.1.0",
         "react-request-fullscreen": "^1.1.2",
         "react-virtualized": "^9.22.5",
-        "uuid": "^3.4.0"
+        "uuid": "^9.0.1"
     },
     "peerDependencies": {
         "@emotion/react": "^11.8.1",
         "@emotion/styled": "^11.8.1",
+        "@hookform/resolvers": "^3.0.0",
         "@mui/icons-material": "^5.5.0",
-        "@mui/lab": "^5.0.0-alpha.72",
+        "@mui/lab": "5.0.0-alpha.169",
         "@mui/material": "^5.5.0",
-        "@hookform/resolvers": "^3.0.0",
         "notistack": "^3.0.0",
-        "react-router-dom": "^6.0.0",
         "react": "^18.0.0",
         "react-dom": "^18.0.0",
-        "react-intl": "^6.0.0",
         "react-hook-form": "^7.41.0",
+        "react-intl": "^6.0.0",
+        "react-router-dom": "^6.0.0",
         "yup": "^1.0.0"
```

Comment:
`peerDependencies` aren't in sync with `dependencies` and `devDependencies` versions

Reply:
Done